patterns/2-structured/discover-your-innersource.md

```diff
@@ -48,18 +31,20 @@ People don't bother looking for internally developed solutions - they might not
 
 Make it easy to find the reusable code.
 
-* Pull in Repo names, descriptions and README.md files into the search engine assuming that a search engine exists. Note: a one-stop-shop kind of search engine for all relevant communication and documentation is difficult to attain. Even when using Github's enterprise offering, people often deploy additional systems like wikis to host content orthogonal to what is in the code repositories, slack channels (or IRC), mailing lists (or even nntp servers), some place to store stuff written down in office formats, search over personal e-mail etc. Several of these systems come with search built-in, but integrating this content in one search box or even just deploying a federated search engine across all sources often doesn't come off-the-shelf.
+* Pull in repo names, descriptions and `README.md` files into the search engine assuming that a search engine exists.
+  * A one-stop-shop kind of search engine for all relevant communication and documentation is difficult to attain.
+  * Even when using GitHub's enterprise offering, people often deploy additional systems like wikis to host content orthogonal to what is in the code repositories, slack channels (or IRC), mailing lists (or even nntp servers), some place to store stuff written down in office formats, search over personal e-mail etc.
+  * Several of these systems come with search built-in, but integrating this content in one search box or even just deploying a federated search engine across all sources often doesn't come off-the-shelf.
 * Implement process change to first check for internal solutions for this problem.
 * Tool with a central view (but people are more inclined to google externally than look internally)
 * Concierge service (guide) to help product people find stuff. Might not scale but could be helpful in the beginning.
-* Need some very visible lighthouse projects that start using inner source components and make positive statements about the inner source program.
-* Establish a common, asynchronous communication channel (e.g., like slack or metamorph or yammer) across team boundaries. This might not scale beyond a certain organization size. It is possible people will start splitting this one channel into multiple channels by topic once traffic gets too high. Note: having one channel for many users of one tool might be considered an anti-pattern because they can't find it unless they already know about it.
+* Need some very visible lighthouse projects that start using InnerSource components and make positive statements about the InnerSource program.
+* Establish a common, asynchronous communication channel (e.g., like slack or metamorph or yammer) across team boundaries.
+  * This might not scale beyond a certain organization size.
+  * It is possible people will start splitting this one channel into multiple channels by topic once traffic gets too high.
```

Comment:
```suggestion
  * When using common chat systems, start with one single chat channel for the InnerSource project, only split into multiple channels by topic once traffic gets too high. This way the channel feels alive. In addition the number of input channels to watch is kept at a minimum for the host team.
```

I think we need a bit of reasoning why one should start with only one communication channel. This might also link to our [Communication Tooling](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/communication-tooling.md) pattern.